rock.osdeps

```diff
@@ -81,7 +81,7 @@ rice:
 qwt5:
     debian,ubuntu:
         default: libqwt5-qt4-dev
-        '20.04,22.04': nonexistent
+        '20.04,22.04,24.04': nonexistent
```

Comment:
This should be switched around, default should be the "latest" osdep (in hope this does not change in future).

please use  default: nonexistent and state os for libqwt5-qt4-dev explicitely